LANDING_PAGE.md

```diff
@@ -0,0 +1,132 @@
+
+Finally, when `Question` is selected, the option to pick one or more specific questions appears. The tool then produces and displays a vertical bar chart which contains frequency information for each of the selected questions in the assignment. This is helpful for comparing the ways in which different terms are utilized within different questions in an assignment.
+
+<img src="resources/images/frequency.png" alt="browser" style="width:100%"/>
+
+### Sentiment Analysis
+
+Sentiment analysis (or opinion mining) is the use of natural language processing, text analysis, computational linguistics, and biometrics to systematically identify, extract, quantify, and study affective states and subjective information. Overall,
+this is a technique to determine whether data is positive, negative, or neutral.
+
+Within the GatorMiner tool, you have the ability to choose `Sentiment Analysis` as an analysis option after the path to the desired reflection documents is submitted.
+
+When the tool runs a Sentiment analysis, on any number of assignments, it provides 3 different options to choose from:
+
+- Overall
+- Student
+- Question
+
+When `Overall` is selected, a scatter plot and a bar chart appear on the screen
+displaying the overall sentiment polarity in, for example, assignment-01 given by the users.
+
+When `Student` is selected, it allows the user to choose a specific student to
+observe. When chosen it shows the sentiment shown by the chosen user with a mini bar graph and a bigger version of that using a histogram. Inside this feature, you can also change the number of plots per row.
+
+Finally, when `Question` is selected, it allows the user to choose a certain question in the drop down menu. When chosen, it shows the user the sentiment the question was given.
+
+<img src="resources/images/sentiment.png" alt="browser" style="width:100%"/>
+
+### Document Similarity
+
+Document similarity analyzes documents and compares text to determine frequency of words between documents.
+
+Within the GatorMiner tool, you have the ability to choose `Document Similarity` as an analysis option after the path to the desired reflection documents is submitted.
+
+In the `Document Similarity` section, you are able to select the type of similarity analysis `TF-IDF` and `Spacy`.
+
+When `TF-IDF` is selected, the application will display a frequency matrix showing the correlation between documents. It does this buy dividing the frequency of the word by the total number of terms in a document.
+
+When `Spacy` is selected, the application will display a drop down named 'Model name' with two options:
+
+- `en_core_web_sm` which is used to produce a correlation matrix for **SMALLER** files. (10mb)
+- `en_core_web_md` which is used to produce a correlation matrix for **LARGER** files. (>10mb)
+
+**Warning exceeding these file limits could cause the program to crash.**
+
+<img src="resources/images/similarity.png" alt="browser" style="width:100%"/>
```

Comment:
I didn't know about this before! It's a good thing to know. I would just like to maybe have a source that can refer to here.